geochem_inverse_optimize.py

```diff
@@ -59,7 +60,9 @@ def cp_log_ratio(a, b: ReciprocalParameter):
     return cp.maximum(a * b.rp, b.p * cp.inv_pos(a))
 
 
-def nx_topological_sort_with_data(G: nx.DiGraph) -> Iterator[Tuple[str, pyfastunmix.SampleNode]]:
+def nx_topological_sort_with_data(
+    G: nx.DiGraph,
+) -> Iterator[Tuple[str, pyfastunmix.SampleNode]]:
```

Comment:
I wish we could sort out what's happening with the line lengths. The line length before wrapping is 98, so I'm guessing your configuration is somehow for a line length of 80.

Reply:
Here's the contents of my `.pre-commit-config.yaml` and a screenshot of the black formatting settings on VSCode. Everything should be being set to 100.

![image](https://user-images.githubusercontent.com/10188895/205019749-15f19ae3-8a6b-483e-96ec-2436318945a1.png)


```repos:
    - repo: https://github.com/ambv/black
      rev: 22.6.0
      hooks:
        - id: black
          language: python
          types: [python]
          args: ["--line-length=100"]
    - repo: https://github.com/PyCQA/isort
      rev: 5.10.1
      hooks:
      - id: isort
        name: Sorting import statements
        entry: bash -c 'isort "$@"; git add -u' --
        language: python
        args: ["--filter-files"]
```